content/arduino-cloud/03.cloud-interface/00.sketches/sketches.md

```diff
@@ -129,7 +129,188 @@ The "Secret" File contains your secret credentials, such as Wi-Fi® network SSID
 
 This file will be visible as a "Secret" tab in the Cloud Editor and is named `arduino_secrets.h`, which is not visible on the Cloud platform.
 
-Note that if you are using the offline IDE / Arduino CLI, you will need to manually create this file. More information is in the **Offline Sketches section** just below.
+Note that if you are using the offline IDE / Arduino CLI, you will need to manually create this file. More information is in the **Offline Sketches section**.
+
+## thingsProperties.h and the NetworkConfigurator
```

Comment:
Not sure if this needs to be in 'code' for the linter to not register it as an issue